### PR TITLE
feat: refresh Inference Server Details page

### PR DESCRIPTION
**Pull request by axel7083**
### What does this PR do?

Implement the @MariaLeonova mock-up for the Inference Server Details page from https://github.com/containers/podman-desktop-extension-ai-lab/issues/1691#issuecomment-2387868931.

### Diff with mock-ups

- Adding `arrow-up-right-from-square`[^1] to the Model as the link is clickable and it was not clear
- Removing the `name` property bellow the model, as it would be repetitive with the model name above
- Removing the `quantisation` as cannot get this information easily.

[^1]: https://fontawesome.com/icons/arrow-up-right-from-square?f=classic&s=solid

### Screenshot / video of UI

**Server Running**

image

**Server Stopped**

image

### What issues does this PR fix or reference?

Fixes https://github.com/containers/podman-desktop-extension-ai-lab/issues/1592

### How to test this PR?

- [x] Existing unit tests ensure no regression

### CODE REVIEW

**Review by MariaLeonova — COMMENTED** [review]
Looks good to me!

**Review by benoitf — APPROVED** [review]

**Review by gastoner — COMMENTED** [review]
When opening details of this model (one on the beginning)

image

I would also add at least some details about the model or don't offer the option to open the model if there are no details

I'm not sure of the tooltip should show under the element (I would imagine that the tooltip would show on right - over the `Access` which should reflect light/dark mode)



https://github.com/user-attachments/assets/faf066f8-5a93-455b-bce7-ff1bda9aa9aa


Otherwise LGTM

**Comment by axel7083** [comment]
> I would also add at least some details about the model or don't offer the option to open the model if there are no details

Thanks for reporting that, however this is models specific, for each model we have a corresponding description (markdown), and it is displayed in here.

We had some project at some point to add `metadata` page to the models, but this has not been going further (https://github.com/containers/podman-desktop-extension-ai-lab/pull/1345).

I still think we should keep our system reactive, and allow redirection between the resources, even tho they might be a bit empty for some of them.

> I'm not sure of the tooltip should show under the element

Yeah I also agree, thanks for the feedback I will update!

**Comment by MariaLeonova** [comment]
@axel7083 do you know why this particular model doesn't have any description? It seems like a big and important one - can we add something?

**Comment by axel7083** [comment]
> @axel7083 do you know why this particular model doesn't have any description? It seems like a big and important one - can we add something?

Not related to this PR; you can open an issue

**Comment by MariaLeonova** [comment]
> > @axel7083 do you know why this particular model doesn't have any description? It seems like a big and important one - can we add something?
> 
> Not related to this PR; you can open an issue

https://github.com/containers/podman-desktop-extension-ai-lab/issues/1881

File: packages/frontend/src/pages/InferenceServerDetails.svelte [hunk 1]
```diff
@@ -218,84 +214,124 @@ function handleOnChange(): void {
...
-                        {service.labels['api']}
-                        <Fa class="ml-2" icon={faPlug} />
-                      </CopyButton>
-                    {/if}
-                  {/if}
+                        <!-- Documentation URL -->
+                        <div class="grow">
+                          {#if 'docs' in service.labels}
+                            <Tooltip tip="Open swagger documentation">
+                              Access
```
Comment (gastoner) [inline packages/frontend/src/pages/InferenceServerDetails.svelte:246]: Access should reflect light/dark theme 


https://github.com/user-attachments/assets/3496a4f6-20ba-46df-bf7d-c937a508e9cb

Also I would say that the tooltip should be visible only when you hover the link `swagger documentation` and not the text `Access`

Comment (axel7083) [reply packages/frontend/src/pages/InferenceServerDetails.svelte]: image

image

Nice catch! Fixed :heavy_check_mark:

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by gastoner — APPROVED** [review]

**Pull request merged**